content/en/docs/Tracing/tempo-walkthrough.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Grafana Tempo tracing
```

Comment:
I think the title here might be better with something like "Tracing Tutorial with Tempo". Something that indicates this isn't an extension of the literal documentation but related.